tests/Controller/EnMarche/Security/AdherentSecurityControllerTest.php

```diff
@@ -47,7 +47,7 @@ public function testAuthenticationIsSuccessful(): void
 
         $crawler = $this->client->followRedirect();
         $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
-        $this->assertSame(1, $crawler->selectLink('Carl')->count());
+        $this->assertSame('Carl', trim($crawler->filter('span.topbar-dropdown--trigger')->first()->text()));
```

Comment:
I think this fails becaus it should be
```suggestion
        $this->assertSame('Carl', trim($crawler->filter('span.js-topbar-dropdown--trigger')->first()->text()));
```
no ?